.github/workflows/jre8-scan.yml

```diff
@@ -1,11 +1,9 @@
 name: Scan jre/8 Docker image
 
 on:
-  pull_request:
-    branches:
-    - master
```

Comment:
I remove this event because the events will be triggered by `OR` logic so either `PR to master` or `files pushed in jre/8/**` can trigger the `scan` job.
Then, the `PR to master` seems a bit redundant.